rincon-orlando/academy-go-q42021
# Initial commit for deliverable one

How to test:
Following APIs are available:

To get them all
`GET http://localhost:8082/pokemons`

To get just one of them
`GET http://localhost:8082/pokemons/1`
`GET http://localhost:8082/pokemons/2`
`GET http://localhost:8082/pokemons/3`

Covers:
- Create an API
- Add an endpoint to read from a CSV file
- The CSV should have any information, for example:
	1,bulbasaur
	2,ivysaur
	3,venusaur
- The items in the CSV must have an ID element (int value)
- The endpoint should get information from the CSV by some field (example: ID)
- The result should be displayed as a response
- Use best practices
- Handle the Errors (CSV not valid, error connection, etc)

Does not cover yet:
- Clean architecture
